File: Using_Elasticsearch.Common/Views/MainScreen/Request/RequestGetFiltersMainScreenView.cs
using Using_Elasticsearch.Common.Enums;
using Using_Elasticsearch.Common.Models;

namespace Using_Elasticsearch.Common.Views.MainScreen.Request
{
    public class RequestGetFiltersMainScreenView
    {
        public FilterName CurrentFilter { get; set; }
        public int Size { get; set; }
        public FiltersModel Filters { get; set; }
    }
}

File: ClientApp/src/app/shared/models/main-screen/request/request-get-filters-main-screen-view.ts
import { FiltersModel } from 'src/app/shared/models/filters-model';

export class RequestGetFiltersMainScreenView {
  currentFilter: number;
  size: number;

  public filters: FiltersModel;

  constructor() {
    this.reset();
  }

  reset(): void {
    this.filters = new FiltersModel();
    this.filters.holidayYear = [];
    this.filters.weekNumber = [];
    this.filters.keyPeriodName = [];
    this.filters.regionName = [null];
    this.filters.parkName = [null];
    this.filters.accommName = [null];
    this.filters.accommTypeName = [null];
    this.filters.accommBeds = [null];
    this.filters.unitGradeName = [null];
    this.filters.responsibleRevenueManager = [null];
  }
}
